vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -1,5 +1,9 @@
 ---
 # tasks file for confluent-install
+- name: Upgrade confluent platform
+  import_tasks: upgrade-confluent.yml
+  when: confluent_major_version is version('6.3', '<')
```

Comment:
The `confluent_major_version` is only `6.2`, wouldn't this upgrade be triggered every single time, until be bump the major version again?  Or should `confluent_major_version` be `6.3`?

Reply:
You are right. I've neglected that and was more focused on upgrading from 5.4.. Thanks @david-oc-miller